IdentityCore/src/cache/MSIDToken.h

```diff
@@ -57,7 +57,7 @@ typedef NS_ENUM(NSInteger, MSIDTokenType)
 - (BOOL)isExpired;
 
 - (instancetype)initWithTokenResponse:(MSIDTokenResponse *)response
-                              request:(MSIDTokenRequest *)request
+                              request:(MSIDRequestParameters *)requestParams
                             tokenType:(MSIDTokenType)tokenType;
```

Comment:
do we need token type here? 
can we do something like the following to determine tokenType from response?

    if (tokenResponse.idToken && tokenResponse.accessToken) {
        // access token
    }
    
    if (tokenResponse.idToken && tokenResponse.refreshToken) {
        // refreshToken token
    }
    
    if (!tokenResponse.idToken) {
        // adfs token
    }

Reply:
Talked offline that token response contains both access token and refresh token, so it won't be possible to check that way. Also, it's good to be explicit about the token type.